ezua/whitelist_disks/README.md

```diff
@@ -0,0 +1,50 @@
+
+# Introduction to Whitelisting disks
+
+A user may have many disks on a node and does not want Data Fabric to use all of them.  A user can choose which disks are used by Data Fabric on each node.  This process is called whitelisting disks.  The disks that will be used on each node can be different. 
+
+The list of disks will be provided in a text file called /mnt/whitelist.  Here is a sample: 
+
+m2-lr1-dev-vm20.mip.storage.hpecorp.net: /dev/sdb  
+m2-lr1-dev-vm21.mip.storage.hpecorp.net: /dev/sdc  
+m2-lr1-dev-vm22.mip.storage.hpecorp.net: /dev/sdb,/dev/sdc  
+m2-lr1-dev-vm23.mip.storage.hpecorp.net: /dev/sdd
+
+The whitelist file can have FQDN or IP addresses.
+
+The file with the list of disks needs to be copied to each worker node before UA is installed.  The file must be called "whitelist" and must be placed in /mnt.  The user may need sudo privileges to place the whitelist file in /mnt.  The push_whitelist bash script is provided to push the whitelist file to each node specified in the file.  It is optional to use this script. The user can distribute the whitelist file themselves.
+
+# Prerequisites
```

Comment:
not clear, at what stage of the UA install, we should run this script

Reply:
I ran the script as my first step, before starting the UI.  But it could be done any time before DF install happens.  Maybe it can be done after the user fills out the UI pages and before they click "submit"?

Or maybe I can say before starting the UI?

Reply:
I added a comment to run the script before starting the UI.